### DISCUSSION TITLE
Trivy security vulnerabilities

### DISCUSSION THREAD

**Discussion by Kartikey-star** [artifacthub/hub#2091] [Q&A] [answered]
Hi, has there been a recent update on trivy workflow on artifact hub because of which the dashboard  shows the low and medium vulnerabilities too in addition to high and critical.

**Comment by tegioz** [comment, marked as answer]
Hi @Kartikey-star 👋

Yes, we update it frequently, a few days after a new release is out. We actually have to upgrade to 0.29.2 IIRC, we'll take care of it soon :)
